### PR TITLE
Add midnight workflow

### PR DESCRIPTION
**Pull request by RostyslavBaldovskyi**
Link: https://github.com/exberry-io/docs/issues/1514

Behavior:
- In repos default branch is develop, meaning scheduler will build develop
- If scheduled build is failed it will create ticket - https://github.com/exberry-io/docs/issues/...
- If ticket was already created, new one will not be created (in scope of 1 repo)
- If build was successful previously opened issue about failed build will be closed
- If closed it will have "Closed due to success build: ${link}" comment
- Ticket name example "Scheduled Build Fail (iam-service)"
- Ticket has a link to the build and 'Bug' label
- Scheduled builds will not do deploys (push)

Note:
Each repo should be updated to have "scheduler" property, it is not inherited from the parent

iam-service PR: https://github.com/exberry-io/iam-service/pull/137

Example:

// Has intentionally broken test (ticket added)
https://github.com/exberry-io/iam-service/actions/runs/11237833810

// Made tests to work again (ticket closed)
https://github.com/exberry-io/iam-service/actions/runs/11237958557

// This task was created and then closed due to recovered build
https://github.com/exberry-io/docs/issues/2542

### CODE REVIEW

**Comment by artem-v** [comment]
@RostyslavBaldovskyi here it's said: 

> Scheduled builds will not do deploys (push)

But after looking at 

> // Made tests to work again (ticket closed)
https://github.com/exberry-io/iam-service/actions/runs/11237958557

It looks like robokit deployed service.

**Comment by artem-v** [comment]
> // This task was created and then closed due to recovered build
https://github.com/exberry-io/docs/issues/2542

Another question: why tasks are being created as they are created by @ronenhamias? Can't we have special user, for example like `exberry-ci` or `exberry-bot`.  

Looping in @nicolaicinali, maybe he has some idea in this regard too.

**Comment by nicolaicinali** [comment]
> > // This task was created and then closed due to recovered build
> > [exberry-io/docs#2542](https://github.com/exberry-io/docs/issues/2542)
> 
> Another question: why tasks are being created as they are created by @ronenhamias? Can't we have special user, for example like `exberry-ci` or `exberry-bot`.
> 
> Looping in @nicolaicinali, maybe he has some idea in this regard too.

Most likely it depends on the GitHub token, especially who is the owner of the token. GH CI uses `ORGANIZATION_TOKEN` GH token. Maybe it was created by Ronen?...
in order to have the right answer we need to understand what steps have been executed when the GH issue have been created and closed.

**Comment by RostyslavBaldovskyi** [comment]
> @RostyslavBaldovskyi here it's said:
> 
> > Scheduled builds will not do deploys (push)
> 
> But after looking at
> 
> > // Made tests to work again (ticket closed)
> > https://github.com/exberry-io/iam-service/actions/runs/11237958557
> 
> It looks like robokit deployed service.

Will check

**Comment by RostyslavBaldovskyi** [comment]
Pushed a fix for robokit to iam-service. 
Will not change cron for now, tomorrow we`ll see if it works. 
It should run at midnight UTC

**Comment by RostyslavBaldovskyi** [comment]
It worked: https://github.com/exberry-io/iam-service/actions/runs/11246029727

**Comment by RostyslavBaldovskyi** [comment]
> we need to understand what steps have been executed when the GH issue have been created and closed.

Steps are defined in file which is changed:
- notify_failure
- close_issue

It uses `secrets.ORGANIZATION_TOKEN`

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -393,58 +394,160 @@ jobs:
...
+                  });
+                  console.log(`Closed issue #${issue.number}`);
+                  issueClosed = true;
+                }
+              }
+            
+              if (!issueClosed) {
+                console.log('No issues found to close');
+              }
+            }
```
Comment (artem-v) [inline .github/workflows/ci.yml]: Let's  have new line at the end. Plz add new line.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -393,58 +394,160 @@ jobs:
...
+
       - name: Rollback release
         if: failure() && github.event_name == 'release'
         run: |
           echo Remove git tag
-          git push origin :refs/tags/${{ github.event.release.tag_name }}
\ No newline at end of file
+          git push origin :refs/tags/${{ github.event.release.tag_name }}
+
+
```
Comment (artem-v) [inline .github/workflows/ci.yml]: Remove one empty line

**Review by artem-v — CHANGES_REQUESTED** [review]

**Review by artem-v — APPROVED** [review]
LGTM

**Pull request merged**